test/rpc_test.py

```diff
@@ -82,9 +74,9 @@ def test_log_rpc_retries_enabled(self, mock_logger):
         self.get_work(fetch_results)
         self.assertEqual([
             mock.call.warning('Failed connecting to remote scheduler %r', 'http://zorg.com', exc_info=True),
-            mock.call.info('Retrying attempt 2 of 3 (max)'),
+            mock.call.info('Retrying attempt 1 of 3 (max)'),
```

Comment:
What caused this different behaviour?

Reply:
This log shows the number of "retry" attempt, but in original one it shows the number of total try (including 1st attempt, not "retry" attempt). So I fixed.

Reply:
I see. With this change, `Retrying attempt n of 3 (max)` would be wrong because `n` has already been tried. If we still prefer log being issued after an attempt, the original counting looks more accurate.

Reply:
@honnix Thanks for your comment! Make sense!
I fixed this. Please check.